src/core/Akka.Persistence.Tests/ReceivePersistentActorTests.cs

```diff
@@ -31,9 +31,9 @@ public void Given_persistent_actor_with_no_receive_command_specified_When_receiv
             //Given
             var pid = "p-1";
             WriteEvents(pid, 1, 2, 3);
-            var actor = Sys.ActorOf(Props.Create(() => new NoCommandActor(pid)), "no-receive-specified");
             Sys.EventStream.Subscribe(TestActor, typeof(UnhandledMessage));
```

Comment:
Need to subscribe to the `EventStream` first, so we don't end up with a race condition behind the scenes.